lib/asyncbox.js

```diff
@@ -87,9 +62,7 @@ function nodeifyAll (promiseyMap) {
 }
 
 function asyncify (fn, ...args) {
-  fn.call(null, ...args).then(() => {}, (err) => {
-    console.error(err.stack);
-  });
+  B.resolve(fn.call(null, ...args)).done();
```

Comment:
`done` takes care of printing the error? or it throws it in the calling context?